### PR TITLE
Add sentry to frontend, add env-settings endpoint

### PR DESCRIPTION
**Pull request by Thunderguts13**
Sentry is implemented in Ember. The DSN gets retrieved via env_settings endpoint. Right now the Sentry-DSN is the only Parameter on this endpoint. Are there any other Environment Variables which could be included?

### CODE REVIEW

File: frontend/app/initializers/env-settings.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import config from "../config/environment";
+
+export function initialize(/* application */) {
+  /* eslint-disable no-undef  */
+  $.getJSON({
+    url: "/api/env_settings",
+    async: false,
```
Comment (Thunderguts13) [inline frontend/app/initializers/env-settings.js:7]: Sentry relies upon the synchronous call to the Backend to receive the dsn in the right moment. The problem is that this Mehthod is depracated. So I'm open for suggestions for a different solution.

File: spec/controllers/api/env_settings_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      sentry_env_vars
+
+      get :index
+
+      expect(json['sentry']).to eq('123456')
+    end
+  end
+end
+
+def sentry_env_vars
```
Comment (TheWalkingLeek) [inline spec/controllers/api/env_settings_controller_spec.rb:21]: maybe rename this to `prepare_sentry_env_vars`? and put it inside a `before(:each)`?

File: frontend/app/initializers/env-settings.js (hunk 1 above)
Comment (TheWalkingLeek) [reply frontend/app/initializers/env-settings.js:7]: Don't really know a different way of handling this...

File: frontend/app/initializers/sentry.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import * as Sentry from '@sentry/browser'
+import * as Integrations from '@sentry/integrations';
+import ENV from "../config/environment";
+import { isPresent } from '@ember/utils';
+
+export function initialize() {
+  if (ENV.environment === "production" && isPresent(ENV.sentryDsn)) {
```
Comment (psunix) [inline frontend/app/initializers/sentry.js:7]: nice !

File: spec/controllers/api/env_settings_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Api::EnvSettingsController do
+  include ControllerHelpers
+
+  before(:each) do
```
Comment (psunix) [inline spec/controllers/api/env_settings_controller_spec.rb:8]: before (without :each) should do it :)

File: spec/controllers/api/env_settings_controller_spec.rb [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    ENV['SENTRY_DSN_FRONTEND'] = '123456'
+  end
+
+  context 'GET index' do
+
+    it 'returns env_settings with keycloak enabled' do
```
Comment (psunix) [inline spec/controllers/api/env_settings_controller_spec.rb:14]: keycloak ?

**Pull request merged**